src/main/java/de/tum/in/www1/artemis/service/ProgrammingExerciseService.java

```diff
@@ -450,7 +450,16 @@ private void setupTestTemplateAndPush(Repository repository, Resource[] resource
 
                     try {
                         Resource[] projectTypeTestFileResources = resourceLoaderService.getResources(projectTypeTemplatePath);
```

Comment:
If we add a method `resourceLoaderService.getExistantResources(projectTypeTemplatePath)` then we can move the lines written bellow into that method and save some space here

Reply:
Good catch. Thanks!